subprojects/core/src/main/java/org/gradle/api/internal/tasks/NonCacheableTaskOutputPropertySpec.java

```diff
@@ -48,8 +48,8 @@ public String getPropertyName() {
     }
 
     @Override
-    public SnapshotNormalizationStrategy getSnapshotNormalizationStrategy() {
-        return parent.getSnapshotNormalizationStrategy();
+    public PathNormalizationStrategy getPathNormalizationStrategy() {
+        return parent.getPathNormalizationStrategy();
     }
```

Comment:
We could, but it seems more consistent to me to return the strategy of the parent. It seemed easy enough to leave it in here.